design/auth.go

```diff
@@ -52,6 +52,15 @@ var _ = a.Resource("logout", func() {
 		a.Response(d.TemporaryRedirect)
 		a.Response(d.InternalServerError, JSONAPIErrors)
 	})
+
+	a.Action("logoutv2", func() {
+		a.Routing(
+			a.GET("/v2"),
+		)
+		a.Description("Logout user")
+		a.Response(d.OK)
+		a.Response(d.InternalServerError, JSONAPIErrors)
+	})
```

Comment:
The `logoutv2` endpoint on fabric8-auth can return 400/401/500 http errors (see https://github.com/fabric8-services/fabric8-auth/blob/59ae2d80ad1c3c6722338fb1048fece4c893ef5d/design/login.go#L131-L137) . Shouldn't we have entries for those in here?

Reply:
No. It doesn't matter. These responses in design is just for goa code generation which we do not use anyway. We just proxy the request and the client will get the same response as Auth returned.

Reply:
That means we can get rid of both the `a.Response(..)` blocks?

Reply:
I didn't check but I think we still need at least InternalServerError for proper error handling if some error happens in the proxy. And yeah.. we could probably get rid of OK.